### ISSUE TITLE
We Need to Get Our Styles Consistent

### ISSUE THREAD

**Issue by Zierman** [atormanen/ICS499_ApplicationServer#13]
I think we should try to conform to [these Python standards](https://www.python.org/dev/peps/pep-0008) as much as reasonable for the python code. 

I'm particularly concerned with naming. I'll be changing names to conform to Python conventions when I see the need. 

Other things we need to get consistent:

- Line ending characters (My IDE lets me use any style, but it shows a warning if they are mixed)
- tabs vs spaces (I like tabs because I have less backspaces to press, but I do see value in spaces)
- single vs double quotations for strings (I am using double at the moment out of habit from C)
- docString style (I've been using Google's style for no particular reason)

@atormanen, I'm generally happy to conform to whatever style you prefer if it doesn't conflict with established standards. The way I see it, the earlier we establish a consistent style preference for our organization, the less work it will down the road.

**Comment by Zierman** [comment]
I was using tabs when spaces are preferred. I'm changing my IDE's settings and correcting any issues I find.

**Comment by Zierman** [comment]
If you notice any other deviations from preferred styles in my code, please don't hesitate to mention it to me so I can become aware and work on fixing my style. 👍

**Comment by atormanen** [comment]
Yes, don't use tabs! Tabs can go die! They make life miserable when we go
to run the code on the server.

We can continue to use the same standard that's already been implemented in
the codebase. I don't like it but no point in changing now.

**Comment by Zierman** [comment]
I don't mind doing the work to change anything. It just might take a little time. I'm definitely planning to rename things to conform to Python's style.

**Comment by Zierman** [comment]
I personally like coding with tabs when it doesn't cause problems because I can edit code quicker with fewer keystrokes (particularly backspace keystrokes), but I also do understand why spaces are generally preferable.
